extra-dev/packages/coq-ltac2/coq-ltac2.dev/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+name: "coq-ltac2"
+version: "0.1"
+maintainer: "Pierre-Marie Pédrot <pierre-marie.pedrot@irif.fr>"
+author: "Pierre-Marie Pédrot <pierre-marie.pedrot@irif.fr>"
+license: "LGPL 2.1"
+homepage: "https://github.com/ppedrot/ltac2"
+dev-repo: "https://github.com/ppedrot/ltac2.git"
+bug-reports: "https://github.com/ppedrot/ltac2/issues"
+build: [
+  [make "COQBIN=\"\"" "-j%{jobs}%"]
```

Comment:
I'm not so sure we can protect our build scripts by polluted user environments.
I mean, this is not the only variable that could make the build fail if incorrectly set...